deploy/kubernetes/README.md

```diff
@@ -0,0 +1,36 @@
+# cAdvisor Kubernetes Daemonset
```

Comment:
Consider adding a link to this from the main cadvisor readme

Reply:
done